sources/PTYTextView.m

```diff
@@ -4831,8 +4833,10 @@ - (void)doCommandBySelector:(SEL)aSelector
         // See comment in -keyDown:
         DLog(@"Rerouting doCommandBySelector from %@ to %@", self, gCurrentKeyEventTextView);
         [gCurrentKeyEventTextView doCommandBySelector:aSelector];
+        [self.delegate keyDown:_keydownEvent];
         return;
     }
+    [self.delegate keyDown:_keydownEvent];
```

Comment:
You're right, I commented on the wrong line. You shouldn't call keyDown: inside the if statement, but after it is OK.